docs/Pipelines/T1_Linear.md

```diff
@@ -43,6 +43,9 @@ On default, cropped images (matrix size 169×208×179, 1 mm isotropic voxels) ar
 generated to reduce the computing power required when training deep learning models.
 Use the option `--uncropped_image` if you do not want to crop the image.
 
+It is also possible to obtain a deterministic result by setting the value of the random
+seed used by ANTs with the option `--random_seed`. Default will lead to a non-determinstic result.
```

Comment:
Just to be sure 😅 

```suggestion
It is also possible to obtain a deterministic result by setting the value of the random
seed used by ANTs with the option `--random_seed`. Default will lead to a non-determinstic result. This option requires ANTs version `2.3.0` onwards.
```

Reply:
No idea, but it won't hurt to put it anyway (even if it is possible with a previous version)